api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -54,6 +54,7 @@ public static boolean isSocketTimeoutException(Throwable e) {
 
   public static RuntimeException convertFirecloudException(ApiException e) {
     log.log(e.getCode() >= 500 ? Level.SEVERE : Level.INFO, "Exception calling FireCloud", e);
+    log.log(Level.INFO, e.getResponseBody());
```

Comment:
I would argue that the `INFO` lines here and on 56 (and below) should probably be `Level.WARN` at a minimum. If we're throwing an exception, I want these logs to stand out in Stackdriver.

Reply:
I would just make this a single log statement:

log.log(... "Exception calling FireCloud: " + e.getResponseBody(), e)

I'm fine with making it WARN for 4xx errors.